simplq/src/components/pages/Admin/Token.jsx

```diff
@@ -65,40 +65,49 @@ function Token(props) {
     // Yet to notify
     notificationButton = (
       <IconButton
-        edge="end"
         color="primary"
+        edge="end"
         aria-label="notify"
         onClick={onNotifyClick}
         onMouseEnter={handleMouseHover}
         onMouseLeave={handleMouseHover}
       >
         {isNotifyHovering ? (
-          <NotificationsActiveIcon fontSize="large" />
+          <NotificationsActiveIcon fontSize="large" className={styles['token-icon']} />
         ) : (
-          <Notifications fontSize="large" />
+          <Notifications fontSize="large" className={styles['token-icon']} />
         )}
       </IconButton>
     );
   }
 
+  const RemoveButton = () => (
+    <div onClick={onDeleteClick} className={styles['token-remove']} aria-label="remove">
+      <p>remove</p>
+    </div>
+  );
+
   return (
-    <div className={styles.token}>
-      <div>
-        <div>{notificationButton}</div>
-        <div>
-          <p className={styles['person-name']}>{name}</p>
-          <p className={styles['token-number']}>
-            Token No:
-            <span className={styles['token-number-value']}>{tokenNumber}</span>
-          </p>
-        </div>
+    <section className={styles.token}>
+      <div className={styles['token-number']}>
+        <p>{tokenNumber < 99 ? `00${tokenNumber}` : tokenNumber}</p>
       </div>
-      <div>
-        <IconButton color="primary">
-          <CheckIcon fontSize="large" onClick={onDeleteClick} />
-        </IconButton>
+      <div className={styles['token-details']}>
+        <div className={styles['token-name-time']}>
+          <p>Join time 11:02 pm</p>
+          <p>{name}</p>
+        </div>
+        <div className={styles['token-operations']}>
+          <div className={styles['token-call-notification']}>
+            <IconButton>
+              <CallIcon className={styles['token-icon']} fontSize="large" />
+            </IconButton>
+            {notificationButton}
+          </div>
```

Comment:
Can you check this div on line 101? I don't see the class `token-call-notification` defined

Reply:
Sorry, I forgot to remove this from the jsx file.